package.json

```diff
@@ -170,7 +170,7 @@
     "@gnosis.pm/safe-core-sdk": "^1.1.1",
     "@gnosis.pm/safe-deployments": "^1.5.0",
     "@gnosis.pm/safe-react-components": "^0.9.0",
-    "@gnosis.pm/safe-react-gateway-sdk": "2.7.5",
+    "@gnosis.pm/safe-react-gateway-sdk": "file:.yalc/@gnosis.pm/safe-react-gateway-sdk",
```

Comment:
will change it to the one from npm once it passed all the necessary checks (review, QA if needed)

Reply:
Lol, I just realized it only exists locally on my computer... I asked Ivan to deploy a new version